extensions/amp-form/0.1/amp-form.js

```diff
@@ -99,6 +150,11 @@ export class AmpForm {
   /** @private */
   installSubmitHandler_() {
     this.form_.addEventListener('submit', e => this.handleSubmit_(e), true);
+    const inputs = this.form_.querySelectorAll('input,select,textarea');
```

Comment:
`blur` doesn't bubble but can be captured - [JSBin demo](http://jsbin.com/hedicuy/edit?html,js,output). `input` does bubble. Filed a bug to fix this in subsequent PR. #4305